### PR TITLE
[webui] auto_off hide auto_off_delay when disabled.

### PR DESCRIPTION
**Pull request by andyblac**

### CODE REVIEW

**Comment by andyblac** [comment]
Screenshot 2021-02-12 at 15 43 24
Screenshot 2021-02-12 at 15 43 35

File: fs_src/index.html [hunk 1]
```diff
@@ -286,8 +286,8 @@ <h1 id="head">Switch</h1>
           <span class="slider round"></span>
         </label>
       </div>
-      <div>
-        <label for="auto_off_delay">Auto off delay:</label>
+      <div class="form-control" id="auto_off_delay_container">
+        <label for="auto_off">Auto off delay:</label>
```
Comment (rojer) [inline fs_src/index.html:290]: input element is still "auto_off", so why change this?

Comment (andyblac) [reply fs_src/index.html]: sorry still learning js, is this now correct https://github.com/mongoose-os-apps/shelly-homekit/pull/490/commits/71c9bff14f63aeb3bb077c31c3ed1fc28872ea37 ?

Comment (rojer) [reply fs_src/index.html]: yes, looks good

**Pull request merged**